linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/oval/shared.xml

```diff
@@ -8,6 +8,7 @@
       <extend_definition comment="Installed OS is RHEL8" definition_ref="installed_OS_is_rhel8" />
       <extend_definition comment="Installed OS is RHCOS4" definition_ref="installed_OS_is_rhcos4" />
       <extend_definition comment="Installed OS is OL7" definition_ref="installed_OS_is_ol7_family" />
+      <extend_definition comment="Installed OS is SLE12" definition_ref="installed_OS_is_sle12" />
```

Comment:
For posterity, pasting the link here with certifications that allow this change to be accepted:

https://www.suse.com/support/security/certifications/